docs/site/Sequence.md

```diff
@@ -217,30 +217,106 @@ namespace.
 
 ### Query string parameters
 
-{% include content/tbd.html %}
+OAI 3.0.x describes the data from a request’s header, query and path in an
+operation specification’s parameters property. In a Controller method, such an
+argument is typically decorated by @param(). We've made multiple shortcuts
+available to the `@param()` decorator in the form of
+`@param.<http_source>.<OAI_primitive_type>`. Using this notation, query string
+parameters can be described as `@param.query.<OAI_primitive_type>`. Here is an
```

Comment:
typo? I think you mean `@param.query.string` =p

Reply:
Good catch :-). Fixed!